### PR TITLE
Toggle k-th Bit 

### LINKED ISSUE #272: Toggle k-th Bit
## 💥 Proposal

Toggle k-th Bit Docs

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

YES

### PR DESCRIPTION
**Pull request by Harikrishnan6336**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

Toggle k-th Bit 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes #272

### CODE REVIEW

File: docs/bit-manipulation/toggle-kth-bit.mdx [hunk 1]
```diff
@@ -0,0 +1,130 @@
+---
+title: Toggle K'th bit
+slug: /bit-manipulation/toggle-kth-bit
+---
+
+## Introduction
+
+Given a number, toggle the `k`'th bit of the given number.
```
Comment (HarshCasper) [inline docs/bit-manipulation/toggle-kth-bit.mdx:8]: Give a better context on what toggling means

File: sidebars.js [hunk 2]
```diff
@@ -185,6 +185,7 @@ module.exports = {
                 "cryptography/burrows-wheeler-transform",
                 "cryptography/affine-cipher",
                 "cryptography/vigenere-cipher",
+
```
Comment (HarshCasper) [inline sidebars.js:188]: Remove this line

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Review by iamrajiv — APPROVED** [review]

File: docs/bit-manipulation/toggle-kth-bit.mdx [hunk 3]
```diff
@@ -0,0 +1,131 @@
+---
+title: Toggle K'th bit
+slug: /bit-manipulation/toggle-kth-bit
+---
+
+## Introduction
+
```
Comment (bhav09) [inline docs/bit-manipulation/toggle-kth-bit.mdx:7]: In the intro section can you also add the concept of right and left shift operator. I think that would be a valuable addition. Rest seems perfect to me.

**Review by bhav09 — CHANGES_REQUESTED** [review]
Reviewed.

File: docs/bit-manipulation/toggle-kth-bit.mdx (hunk 3 above)
Comment (Harikrishnan6336) [reply docs/bit-manipulation/toggle-kth-bit.mdx:7]: Done

**Pull request merged**